<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ICHCStackedSettingsFrame</class>
 <widget class="QFrame" name="ICHCStackedSettingsFrame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>318</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <property name="frameShape">
   <enum>QFrame::NoFrame</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Plain</enum>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <widget class="QLabel" name="currentPageLabel">
     <property name="text">
      <string>Current page</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QToolButton" name="page0ToolButton">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>80</width>
       <height>48</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>80</width>
       <height>48</height>
      </size>
     </property>
     <property name="text">
      <string>Page 0</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <attribute name="buttonGroup">
      <string>buttonGroup_2</string>
     </attribute>
    </widget>
   </item>
   <item row="1" column="1" rowspan="4">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Sequence</string>
       </property>
       <layout class="QFormLayout" name="formLayout">
        <property name="horizontalSpacing">
         <number>15</number>
        </property>
        <property name="verticalSpacing">
         <number>15</number>
        </property>
        <item row="0" column="0">
         <widget class="QCheckBox" name="xzyCheckBox">
          <property name="text">
           <string>X-Z-Y</string>
          </property>
          <attribute name="buttonGroup">
           <string>buttonGroup</string>
          </attribute>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QCheckBox" name="yxzCheckBox">
          <property name="text">
           <string>Y-X-Z</string>
          </property>
          <attribute name="buttonGroup">
           <string>buttonGroup</string>
          </attribute>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QCheckBox" name="zxyCheckBox">
          <property name="text">
           <string>Z-X-Y</string>
          </property>
          <attribute name="buttonGroup">
           <string>buttonGroup</string>
          </attribute>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QCheckBox" name="yzxCheckBox">
          <property name="text">
           <string>Y-Z-X</string>
          </property>
          <attribute name="buttonGroup">
           <string>buttonGroup</string>
          </attribute>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="1">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Lattice</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Step</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QCheckBox" name="xRPCheckBox">
         <property name="text">
          <string>X RP</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="ICLineEditWithVirtualNumericKeypad" name="xRPLatticeLineEdit">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>32</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="ICLineEditWithVirtualNumericKeypad" name="xRPStepLineEdit">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>32</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QCheckBox" name="yRPCheckBox">
         <property name="text">
          <string>Y RP</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="ICLineEditWithVirtualNumericKeypad" name="yRPLatticeLineEdit">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>32</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="ICLineEditWithVirtualNumericKeypad" name="yRPStepLineEdit">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>32</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QCheckBox" name="zRPCheckBox">
         <property name="text">
          <string>Z RP</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="ICLineEditWithVirtualNumericKeypad" name="zRPLatticeLineEdit">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>32</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="3" column="2">
        <widget class="ICLineEditWithVirtualNumericKeypad" name="zRPStepLineEdit">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>32</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="1" column="3">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>mm</string>
         </property>
        </widget>
       </item>
       <item row="2" column="3">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>mm</string>
         </property>
        </widget>
       </item>
       <item row="3" column="3">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>mm</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <widget class="QToolButton" name="page1ToolButton">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>80</width>
       <height>48</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>80</width>
       <height>48</height>
      </size>
     </property>
     <property name="text">
      <string>Page 1</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <attribute name="buttonGroup">
      <string>buttonGroup_2</string>
     </attribute>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QToolButton" name="page2ToolButton">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>80</width>
       <height>48</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>80</width>
       <height>48</height>
      </size>
     </property>
     <property name="text">
      <string>Page 2</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <attribute name="buttonGroup">
      <string>buttonGroup_2</string>
     </attribute>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QToolButton" name="page3ToolButton">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>80</width>
       <height>48</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>80</width>
       <height>48</height>
      </size>
     </property>
     <property name="text">
      <string>Page 3</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <attribute name="buttonGroup">
      <string>buttonGroup_2</string>
     </attribute>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ICLineEditWithVirtualNumericKeypad</class>
   <extends>QLineEdit</extends>
   <header>iclineeditwithvirtualnumerickeypad.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup_2"/>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
